src/components/Categories.js

```diff
@@ -1,15 +1,23 @@
-import React from 'react';
-import ProductCategoriesData from '../mocks/en-us/product-categories.json';
+import React, { useContext } from 'react';
 import { Category, CategoriesContainer } from './Categories.styles';
+import { useApi } from '../utils/hooks/useApi';
+import ThemeContext from '../context/ThemeContext';
+
+export default function Categories() {
+  const response = useApi('category', 10, 1);
+
+  const ThemeCtx = useContext(ThemeContext);
 
-export default function Categories({ showMenu }) {
   return (
     <>
-      {showMenu && (
+      {ThemeCtx.showMenu && (
         <CategoriesContainer>
-          {ProductCategoriesData.results.map((category) => {
-            return <Category key={category.id}>{category.data.name}</Category>;
-          })}
+          {response?.data?.results &&
+            response.data.results.map((category) => {
```

Comment:
Haciendo lo anterior ahora podemos dejar este código más reducido y legible.
```suggestion
          {categories &&
            categories.map((category) => {
```